arm-cognitiveservices/2017-04-18/swagger/cognitiveservices.json

```diff
@@ -0,0 +1,1071 @@
+      "post": {
+        "tags": [ "CognitiveServicesAccounts" ],
+        "operationId": "CheckSkuAvailability_List",
+        "description": "Check availalable SKUs.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/locationParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "x-ms-client-flatten": true,
+            "schema": {
+              "$ref": "#/definitions/CheckSkuAvailabilityParameter"
+            },
+            "description": "Check SKU Avaiilablity POST body."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "#/definitions/CheckSkuAvailabilityResultList"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.CognitiveServices/locations/{location}/updateAccountsCreationSettings": {
+      "post": {
+        "tags": [ "CognitiveServicesAccounts" ],
+        "operationId": "CognitiveServices_UpdateResourceKindSettings",
+        "description": "Update resource kind settings.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/locationParameter"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "x-ms-client-flatten": true,
+            "schema": {
+              "$ref": "#/definitions/CognitiveServicesResourceKindSettingUpdateParameter"
+            },
+            "description": "Update Cognitive Services resource kind settings."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.CognitiveServices/locations/{location}/accountsCreationSettings/{creationSettingName}": {
+      "get": {
+        "tags": [ "CognitiveServicesAccounts" ],
+        "operationId": "AccountsCreationSettings_List",
+        "description": "Read resource kind settings.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/locationParameter"
+          },
+          {
+            "name": "creationSettingName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "enum": [ "current" ],
+            "description": "Accounts creation setting name, now only support \"current\".",
```

Comment:
are you planning on adding other values? If there's only one value in the enum on a required parameter, then it's value will be taken as a constant (https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#single-value-enum-as-a-constant) 
You could just write that value in the path as a constant, if you're not planning on adding more values.